patches/jszip+3.10.1.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/node_modules/jszip/dist/jszip.js b/node_modules/jszip/dist/jszip.js
+index 60fbb41..9dda434 100644
+--- a/node_modules/jszip/dist/jszip.js
++++ b/node_modules/jszip/dist/jszip.js
+@@ -11399,10 +11399,6 @@ module.exports = ZStream;
+     var registerImmediate;
+ 
+     function setImmediate(callback) {
+-      // Callback can either be a function or a string
+-      if (typeof callback !== "function") {
+-        callback = new Function("" + callback);
+-      }
```

Comment:
We never pass a string to `setImmediate`, so it's safe to remove.

The changes below in `jszip.min.js` are the same, but the diff looks weird because it's a single-line file 🙃